### PR TITLE
docs: Add process - "How to create issue (templates edition)"

### LINKED ISSUE #18: [DOCS] Add process/guide - "How to create issue (templates edition)"
### Changes/Action required
 
 - Add a complete process of **"How to create issue templates"**, steps will contain a text, screenshot, Video (it can be a screen recording to show the steps), and GIF (if needed)
 
- Path of the file in which **steps** need to be added - [`open-source-with-pradumna/pages/How-to/guide/raise-issue-template.md`](https://github.com/Pradumnasaraf/open-source-with-pradumna/blob/main/pages/How-to/guide/raise-issue-template.md)

---

If you have any suggestions feel free to Open an [Issue](https://github.com/Pradumnasaraf/open-source-with-pradumna/issues)

#### This addition of the documentation will get added and hyperlinked in [`How-to/README.md`](https://github.com/Pradumnasaraf/open-source-with-pradumna/tree/main/pages/How-to) and also get hosted on the website https://Opensource.pradumnasaraf.co

Screenshot from 2022-05-15 14-11-21

### PR DESCRIPTION
**Pull request by Kanika637**
## 🛠️ Fixes Issue

Added the whole procedure of, "How to create issue (templates edition)""

## ✔️ Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## 📷 Screenshots
issue#18 1
issue#18 2
issue#18

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

### CODE REVIEW

File: pages/How-to/guide/adding-issue-template.md [hunk 1]
```diff
@@ -1,3 +1,114 @@
...
+## Let's check all of them one by one
+
+</br>
+
+### 🐛 Bug
+
+
+</br>
+
+-> If you find any bug in the repository feel free to click on `Get Started` and you will get a template in which you will get a template in which you will write a crisp title of the bug and then in the description, ellaborate the bug.
```
Comment (Jai-Solania-29) [inline pages/How-to/guide/adding-issue-template.md:50]: Hey @Kanika637 there is a typo at the last of this line of code. ( Replace ellaborate with elaborate )

**Review by Jai-Solania-29 — COMMENTED** [review]
Please go through the suggestion made in highlighted area. 
Otherwise everything looks good to me 👍.

File: pages/How-to/guide/adding-issue-template.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Kanika637) [reply pages/How-to/guide/adding-issue-template.md:50]: Thanks for the review @Jai-Solania-29 , will surely correct that typo.

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey! Kanika, thank you so much for your PR and effort!.

**There is some change required:**

 - Instead of storing the Image in the repo physically and using paths to define them, we use `githubusercontent` link once (Which means you can drag and drop the image in the Markdown file it will do rest of this for you)
 
 If you need any help feel to reach out.

**Comment by Kanika637** [comment]
@Pradumnasaraf , I actually tried doing the drag and drop thing but, the image gets opened in a new file in the editor (VS code)

**Comment by Pradumnasaraf** [comment]
> @Pradumnasaraf , I actually tried doing the drag and drop thing but, the image gets opened in a new file in the editor (VS code)

Yes, Kanika you won't be able to do that on VsCode as it's the functionality of Github. You can do the changes here.

**Comment by Kanika637** [comment]
I have made the changes @Pradumnasaraf 👍

**Comment by Pradumnasaraf** [comment]
Hey, @Kanika637 I just noticed you have done a small mistake, you added a guide for "How to create an issue with different templates" not for "how to create the templates for the issue". Yesterday's image was not opening so didn't pay much attention to that.😅


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Tags:** Not reusable

**Comment by Kanika637** [comment]
What to do now? @Pradumnasaraf 😅

**Comment by Pradumnasaraf** [comment]
> Okay @Pradumnasaraf will do that+1

Thank you,
Also can you please remove the extra line spacing (one is sufficient) because the prettier will throw an error, and also in `<br/>` tags remove trailing `/`, as Hosted markdown doesn't understand that and will throw an error.

**Comment by Kanika637** [comment]
I have done all the required changes👍 @Pradumnasaraf

**Review by Pradumnasaraf — APPROVED** [review]
FINALLY!. It's time to merge :) :rocket: 

Thank you so much for your patience and effort, Kanika!! 
(Also, there was some minor refactors left, which I did on your behalf)
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome&nbsp; | &nbsp;**Tags:** Maintainable

**Comment by Kanika637** [comment]
Thanks for the guidance @Pradumnasaraf 😀

**Comment by Pradumnasaraf** [comment]
> Thanks for the guidance @Pradumnasaraf 😀

Awesome!!

Your changes are now live on the website link - https://opensource.pradumnasaraf.co/#/pages/How-to/README

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Pull request merged**